test/statetest/statetest_loader.cpp

```diff
@@ -345,4 +346,30 @@ StateTransitionTest load_state_test(std::istream& input)
 {
     return json::json::parse(input).get<StateTransitionTest>();
 }
+
+void validate_deployed_code(const state::State& state, evmc_revision rev)
```

Comment:
Looks good except I'd put this function in the `statetest_runner.cpp`. It can be `static` then.

Reply:
It cannot be there because it's used in tests too. `statetest_runner.cpp` symbols jest included only executable not library. According to CMakeList